--- ./deps/exla/Makefile	2025-06-23 13:36:52
+++ ./deps/exla/Makefile	2025-06-23 14:09:21
@@ -27,6 +27,7 @@
 # Note: this is on :xla 0.5.0 -- things can change with later versions
 CFLAGS = -fPIC -I$(ERTS_INCLUDE_DIR) -I$(XLA_INCLUDE_PATH) -Wall -Wno-sign-compare \
 	 -Wno-unused-parameter -Wno-missing-field-initializers -Wno-comment \
+	 -Wno-missing-template-arg-list-after-template-kw \
 	 -std=c++17 -w -DLLVM_VERSION_STRING=
 
 NVCCFLAGS = -shared -Xcompiler -fPIC
